Finalizing retrospective

ejmockler opened the pull request:
Adding responsiveness to upper portion from winners component

TODO:

- [x] Hyperlink winner cards to Devpost
- [ ] Add sponsor button to hero

- [ ] Update hamburger & navbar links with organizer applications
- [ ] Proofread written content

- [ ] All components responsive

ilopezro committed:
added mlh banner

ilopezro committed:
added scss for navbar

ilopezro committed:
added navbar view

ilopezro committed:
added mlh banner

ilopezro committed:
added right backdrop

ilopezro committed:
refactored code

ilopezro committed:
added more styling to navbar

ilopezro committed:
added backdrop + main hero page

ilopezro committed:
part of code refactoring

ilopezro committed:
part of code refactoring

ilopezro committed:
left pillars added

ejmockler committed:
reuse old backdrop components

ejmockler committed:
refactored hero title area for postlanding & updated hero background

ilopezro committed:
added more styling to navbar

ilopezro committed:
added new route to homepage

ilopezro committed:
Merge branch 'il/hero-postLanding' of https://github.com/CruzHacks/cruzhacks-2020-website into il/hero-postLanding

ejmockler committed:
used calc to make hero responsive to tab-land

ejmockler committed:
started share component

ejmockler committed:
fixed pillar height

ejmockler committed:
temporary CSS for share section

ejmockler committed:
share responsive to tab-land; still needs slack copy link functionality

ejmockler committed:
fixed svg color & button size

ejmockler committed:
share component responsive

ejmockler committed:
updated site description

ejmockler committed:
added CTA with reusable button component

ejmockler committed:
dropped in sponsors

ejmockler committed:
dropped in sponsors

ilopezro committed:
removed unnecesary countdown

ilopezro committed:
removed unnecesary components

ilopezro committed:
removed unnecesary countdown

ilopezro committed:
redid changes

ilopezro committed:
added init about component

ilopezro committed:
added about component to base scss and also to vie w

ilopezro committed:
Merge branch 'il/hero-postLanding' of https://github.com/CruzHacks/cruzhacks-2020-website into il/hero-postLanding

ilopezro committed:
added about section with styling

ilopezro committed:
reused stats from OG page

ilopezro committed:
imported css to post landing folder

ilopezro committed:
added info section

ilopezro committed:
added more responsive cscc to info section

ilopezro committed:
added key property to cta button

ejmockler committed:
setup backdrop for photos component

ejmockler committed:
photo component half done

ilopezro committed:
added project section view

ilopezro committed:
added projects backdrop

ilopezro committed:
added styling

ilopezro committed:
Merge branch 'il/hero-postLanding' of https://github.com/CruzHacks/cruzhacks-2020-website into il/hero-postLanding

ilopezro committed:
changed an attribute to get rid of console error

ilopezro committed:
added more reponsivness to projects sectoin

ejmockler committed:
added fixed purple waves (that are repeatable for animation)

ejmockler committed:
fixed uneven wave

ilopezro committed:
added winners section WIP

ilopezro committed:
added box view

ilopezro committed:
added linkedin links view

ilopezro committed:
added INIT styling

ejmockler committed:
added photo tint/hover animation

ejmockler committed:
Merge branch 'il/hero-postLanding' of https://github.com/CruzHacks/cruzhacks-2020-website into il/hero-postLanding

ilopezro committed:
added remaining winners and links

ilopezro committed:
refactored code to allow for box to move from left and right

ilopezro committed:
added winner images

ilopezro committed:
added images to boxes

ilopezro committed:
added styling to images

ejmockler committed:
added news backdrop

ejmockler committed:
boilerplated styles; news cards todo

ejmockler committed:
rendered news info

ejmockler committed:
added news card style-- thumbnail & text WIP

ejmockler committed:
news header & background responsive

ejmockler committed:
lower quarter components responsive

ejmockler committed:
fixed anchor hierarchy in dom

ejmockler committed:
added hover animation to news cards

ejmockler committed:
fixed waves alignment

ejmockler committed:
fixed projects button centering

ejmockler committed:
adjusted animation

ejmockler committed:
aligned animation timing

ejmockler committed:
adjusted animation for mobile

ejmockler committed:
share button copy onclick working, tooltip WIP

ilopezro committed:
refactored key component to get rid of error messge

ejmockler committed:
aded basic tooltip functionality

ejmockler committed:
fixed tooltip message glitch

ejmockler committed:
share, cta, and sponsors responsive

ejmockler committed:
added photo button

ejmockler committed:
centered photo button absolutely using a ruler

ejmockler committed:
lower third responsive to mobile

ejmockler committed:
repositioned project ttitle

ejmockler committed:
tweaked sponsor logo alignments

ejmockler committed:
properly aligned info text to center

ejmockler committed:
share tooltip working/added different share icon

ejmockler committed:
refactored share responsiveness

ejmockler committed:
added font to tooltip

ejmockler committed:
applied height to fill linkedin containers and box containers, enabling them to stack

ejmockler committed:
used nth-child selector to set proper flex-wrap direction for antiparallel flexbox order

ghost commented:
<img src="https://www.deepcode.ai/icons/green_check.svg" width= "50px" align= "left"/> Congratulations :tada:. DeepCode [analyzed](https://www.deepcode.ai/app/gh/CruzHacks/cruzhacks-2020-website/791bc0d7c0dfa799a84fd9dec01754b72a10a148/CruzHacks/cruzhacks-2020-website/90f1b13a831dd74110543092d7855d0a66f3732b/_/%2F/code/?utm_source=gh_review) your code in 1.118 seconds and we found no issues. Enjoy a moment of no bugs :sunny:.

##### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/CruzHacks/cruzhacks-2020-website/791bc0d7c0dfa799a84fd9dec01754b72a10a148/CruzHacks/cruzhacks-2020-website/90f1b13a831dd74110543092d7855d0a66f3732b/_/%2F/code/?utm_source=gh_review)
---

ejmockler committed:
added padding-bottom to winner containers

padding-bottom gives enough spacing between rows, and is responsive through the calc function.

ejmockler committed:
added padding

git also lets you add a description with a second -m string, just found this out

ejmockler committed:
Merge branch 'em/retrospective' of https://github.com/CruzHacks/cruzhacks-2020-website into em/retrospective

ejmockler committed:
increased mobile news card font size

ejmockler committed:
added hover effect to winner boxes

also positioned category title in linkedin winners box

ejmockler committed:
made winner linkedin profile margin-top constant

responsiveness will be handled with breakpoints here instead of calc; margin values work better as constants since no time for math

ejmockler committed:
set max-width of winner container for desktop

ejmockler committed:
positioned winner banner text

ejmockler committed:
added font styles for winner box title

ejmockler committed:
winners component responsive to mobile

ejmockler committed:
winner component margins now responsive

ejmockler committed:
fixed winner title centering

ejmockler committed:
slight adjustment to winner box letter styles

ejmockler committed:
finished responsiveness on winner component

also added font-family for winner banner text

ejmockler committed:
refined linkedin link margin responsiveness

ejmockler committed:
added breakpoint to center text after winner component column wraps

ejmockler committed:
centered linkedin logos within buttons on winner component

cancelled out 0.25vh hover transform, which affects the logo img position within the anchor link

ejmockler committed:
added devpost links to winner boxes

changed winner box padding-bottom to margin-bottom so box height does not allow link to be clickable in empty space

ejmockler committed:
winners component good to go

ejmockler committed:
tweaked linkedin logo margins for winners

ejmockler committed:
added responsive styles to projects title

ejmockler committed:
turned project button into flexbox to center button text

ejmockler committed:
added text container to winner bottom box so text is relative across screens

ejmockler committed:
refined winner box text container width

ejmockler committed:
projects button responsive & added hover effect

ejmockler committed:
brittle: added padding-bottom to project button to lengthen container

this is brittle because project container height should be set directly, but the prop is not setting properly

ejmockler committed:
added breakpoints to scale project component on large screens

also added title font for info component

ejmockler committed:
info component responsive (relative to winner component)

ejmockler committed:
included right about backdrop in description container for responsiveness

ejmockler committed:
changed rightside about backdrop SVG width prop to viewbox

ejmockler committed:
set right backdrop width prop to auto

ejmockler committed:
hiding top pillar in right backdrop on tab-land

ejmockler committed:
about responsive to tab-port

ejmockler committed:
fixed about backdrop SVG rendering issue by reusing the old PNG code

ejmockler committed:
added temporary /retrospective route

ejmockler committed:
fixing console warnings, cleaning up loose declarations

ejmockler committed:
fixing console warnings, cleaning up loose declarations
